### ISSUE TITLE
Invalid stream generated

### ISSUE THREAD

**Issue by jay58** [alexandercerutti/passkit-generator#1]
I have following pass structure in my code:
examplepass.pass (folder)
> icon.png
> icon@2x.png
> logo.png
> logo@2x.png
> pass.json

keys (folder)
> signerCert.pem
> signerKey.pem
> wwdr.pem

`const Passkit = require("passkit-generator");`
`const fs = require('fs');`
`let pass = new Passkit.Pass({`
`    model: "./examplepass",`
`    certificates: {`
`        wwdr: "./keys/wwdr.pem",`
`        signerCert: "./keys/signerCert.pem",`
`        signerKey: {`
`            keyFile: "./keys/signerKey.pem",`
`           passphrase: "mac001"`
`        }`
`    },`
`    overrides: {},`
`    shouldOverwrite: true`
`});`
`pass.generate()`
`    .then(stream => {`
`        var file = fs.createWriteStream("examplepass.pkpass")`
`        stream.on('finish', () => {`
`            console.log('done')`
`        })`
`        stream.on("error", function (error) {`
`            console.error(error);`
`            process.exit(1);`
`        })`
`        stream.pipe(file);`
`    })`
`    .catch(err => {`
`        console.log("Error: " + err)`
`    });`

Everything is working fine and I am not getting any error. But when i try to open the pass it says unable to open the pass.

Same example structure, If I try with [utility](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/PassKit_PG/YourFirst.html#//apple_ref/doc/uid/TP40012195-CH2-SW23) here then it is working fine and pass is created.

**Comment by alexandercerutti** [comment]
Can you please open "Console" application in `/Applications/Utility`, filter for "Passkit" (write it in search) and then try again to open the stream? You'll get a more detailed error.

**Comment by alexandercerutti** [comment]
@jay58 did you solve the problem?

**Comment by jay58** [comment]
Hi @alexandercerutti sorry for replying late.
The file created is of 20KB.
I can't find any error log in console.

**Comment by alexandercerutti** [comment]
Can you please send me the resulting pass so I can see? You can drop it here?

**Comment by alexandercerutti** [comment]
@jay58 dude, do you still need help? If so, please be sure to receive notifications. Thank you.

**Comment by jennysharps** [comment]
Hi @alexandercerutti I am getting this same issue and when I go to Console I see the following:
<img width="823" alt="screenshot 2018-10-30 at 10 33 36" src="https://user-images.githubusercontent.com/1467161/47712376-7b007000-dc2f-11e8-91c0-aabf74571366.png">

The resulting file is here (I had to change it to .zip extension to upload it): 
[test.zip](https://github.com/alexandercerutti/passkit-generator/files/2529201/test.zip)

**Comment by alexandercerutti** [comment]
@jennysharps Hi! Well, I've to investigate a bit into this. I'll do it later (I'm unable to this right now) and reply to you asap.

Meanwhile, can you send me your chunk of code? It may be useful. Thank you!

**Comment by jennysharps** [comment]
Hi @alexandercerutti thanks for your quick response, it was actually that my certificate setup was not done properly.

For anybody reading this, if you install your certificate & it doesn't ask you to add it to keychain, it won't install properly & you won't be able to export it as a .p12 file.  I exited Keychain Access app and tried to install my certificate again, and it took me through a completely different flow than it did the first time, which resolved my problem & allowed me to export as a .p12 file as per the instructions.

Thanks!

**Comment by alexandercerutti** [comment]
Awesome! @jennysharps 😄 I'm going to close this as I assume the original sender gave up or resolved without giving any explanation and you resolved too.

**Comment by jay58** [comment]
Hi @alexandercerutti, Sorry being late and not much attentive to this issue, I got into same problem of certificates which I resolved it with some other method. Basically the certificates that I used didn't worked with any passkit generator and then I tried with different ssl commands to create it and it worked. The mistake was in the ssl command that I got from the original docs to create certificates. I have been far from connectivity from last month so wasn't able to reply you either. Sorry and Thank you.
